src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.health = health;
+    this.name = name;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
```

Comment:
you can save here animals name, it will be easier )

Reply:
I actually don't realise how to implement it 
Animal.alive.push(this.name) ?

Reply:
now you push this
`Animal.alive.push(this);`
but you can push name that you receive
`Animal.alive.push(name);`